src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -717,21 +765,62 @@ void ResumeCtrlImpl::AddSubscriptions(
 bool ResumeCtrlImpl::CheckIgnCycleRestrictions(
     const smart_objects::SmartObject& saved_app) {
   LOG4CXX_AUTO_TRACE(logger_);
-  bool result = true;
 
   if (!CheckDelayAfterIgnOn()) {
     LOG4CXX_INFO(logger_, "Application was connected long after ign on");
-    result = false;
+    return false;
   }
 
-  if (!DisconnectedJustBeforeIgnOff(saved_app)) {
-    LOG4CXX_INFO(logger_, "Application was dissconnected long before ign off");
-    result = false;
+  if (!CheckDelayBeforeIgnOff(saved_app)) {
+    LOG4CXX_INFO(logger_, "Application was disconnected long before ign off");
```

Comment:
@AByzhynar rename to debug

Reply:
@LuxoftAKutsan Done. Sorry, commit squashed to old one.